app/webpack.config.js

```diff
@@ -35,7 +35,8 @@ fs.copySync(
 );
 
 // ensure all schemas are statically compiled
-const schemaDir = path.resolve(buildDir, './schemas');
+// fix windows path issue, use / as path separator instead of \
+const schemaDir = path.resolve(buildDir, './schemas').replace(/\\/g, '/');
```

Comment:
perhaps we use [`path.posix`](https://nodejs.org/api/path.html#path_path_posix) everywhere? There will be more of these as we add things like static files, federated extensions, etc.

Reply:
I tried [path.posix](https://nodejs.org/api/path.html#path_path_posix), the result of `path.resolve` use posix path separator indeed, however it reduced the driver letter. So [path.posix](https://nodejs.org/api/path.html#path_path_posix) will not fix this problem here.

```js
> buildDir = './build'
'./build'
> require('path').resolve(buildDir, './schemas')
'D:\\code\\python\\jupyterlite\\build\\schemas'
> require('path/posix').resolve(buildDir, './schemas')
'/code/python/jupyterlite/build/schemas'
>
```